src/components/table-of-contents/example-table-of-contents-related-links-with-button.njk

```diff
@@ -0,0 +1,62 @@
+{% from "components/table-of-contents/_macro.njk" import onsTableOfContents %}
+
+{{-
+    onsTableOfContents({
+        "title": 'Contents',
+        "ariaLabel": 'Pages in this guide',
+        "skipLink": {
+            "url": "#0",
+            "text": 'Skip to guide content'
+        },
+        "lists": [
+            {
+                "itemsList": [
+                    {
+                        "url": '#0',
+                        "text": 'Overview',
+                        "current": true
+                    },
+                    {
+                        "url": '#0',
+                        "text": 'Who should take part and why'
+                    },
+                    {
+                        "url": '#0',
+                        "text": 'How your information is used'
+                    },
+                    {
+                        "url": '#0',
+                        "text": 'The 2019 Census Rehearsal'
+                    },
+                    {
+                        "url": '#0',
+                        "text": 'Online census'
+                    }
+                ]
+            }
+        ],
+        "relatedLinksTitle": 'Related publications',
+        "relatedLinksList": [
+            {
+                "itemsList": [
+                    {
+                        "url": '#0',
+                        "text": 'Example publication title'
+                    },
+                    {
+                        "url": '#0',
+                        "text": 'Example publication title 2'
+                    },
+                    {
+                        "url": '#0',
+                        "text": 'Example publication title 3'
+                    }
+                ]
+            }
+        ],
```

Comment:
Dont have a stron opinion either way on this one but we could have a related links object that has the list and title inside it. See what you think

Reply:
I dont think we have an example of this currently. I thought you might be able to do something like this (https://dev.to/giulia_chiola/add-items-to-an-array-in-nunjucks-482e) but if it is proving difficult then maybe we will have to leave it as it is. Its not a big issue because it will be documented with an example

Reply:
Ah excellent that method worked, thank you! Updated now.